htdocs/fontawesome

```diff
@@ -0,0 +1 @@
+./../node_modules/@fortawesome/fontawesome-free/
```

Comment:
```suggestion
../node_modules/@fortawesome/fontawesome-free/
```
Do you need the first `./`? Seems redundant

Reply:
no, probably not

Reply:
@johnsaigle turns out we do need it